streaming/python/runtime/failover.py

```diff
@@ -0,0 +1,30 @@
+class Barrier:
```

Comment:
The python filename is `failover.py`, but `Barrier` and `OpCheckpointInfo` are defined, very confusing.

Reply:
It's python module style, @chaokunyang suggest us to do this.